contracts/DecentHats.sol

```diff
@@ -0,0 +1,324 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {Enum} from "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+import {IAvatar} from "@gnosis.pm/zodiac/contracts/interfaces/IAvatar.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC6551Registry} from "./interfaces/IERC6551Registry.sol";
+import {IHats} from "./interfaces/hats/full/IHats.sol";
+import {LockupLinear, Broker} from "./interfaces/sablier/full/types/DataTypes.sol";
+import {DecentAutonomousAdmin} from "./DecentAutonomousAdmin.sol";
+import {IHatsModuleFactory} from "./interfaces/hats/full/IHatsModuleFactory.sol";
+import {IHatsElectionEligibility} from "./interfaces/hats/full/IHatsElectionEligibility.sol";
+import {ModuleProxyFactory} from "@gnosis.pm/zodiac/contracts/factory/ModuleProxyFactory.sol";
+
+contract DecentHats {
+    string public constant NAME = "DecentHats";
```

Comment:
Same comments here. Instead of this NAME variable, I'd like to see ERC165 support built into this contract.

Reply:
We also probably just don't even need this NAME string in the contract either. Like who cares, yanno? Idk why I added it in DecentHats_0_1_0.

Reply:
fixed